keps/sig-node/277-ephemeral-containers/README.md

```diff
@@ -831,16 +946,30 @@ _This section must be completed when targeting beta graduation to a release._
   ephemeral containers. We'll investigate whether it's possible to add ephemeral
   containers to these existing tests.
```

Comment:
Any luck with this? Or could you do it in 1.25?

Reply:
No, I didn't get these added last time and now it looks like this link is broken. I'm going to add node conformance tests for 1.25, so I can take another look here. I'll ask around in slack if I can't find the right place to put it.